spec/crowbar/client/request/proposal/edit_spec.rb

```diff
@@ -34,7 +34,7 @@
     end
 
     let!(:params) do
-      {}
+      "{}"
```

Comment:
Can we add a real spec here so that we do not regress again? E.g. check that we're not re-encoding valid json?